CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [0.3.22](https://github.com/ASFHyP3/hyp3-docs/compare/v0.3.20...v0.3.21)
 
 ### Changed
-* Corrected the attribution for Figure 1 in the [InSAR Product Guide](docs/guides/insar_product_guide.md)
+* Corrected the attribution for figures in the [InSAR Product Guide](docs/guides/insar_product_guide.md) and the [Introduction to SAR](docs/guides/introduction_to_sar.md)
```

Comment:
@jhkennedy I combined the changes in the changelog entry. Do you think it's better to keep them as separate bullets?

Reply:
No, this looks great!